src/api/services/Authentication.js

```diff
@@ -0,0 +1,27 @@
+import { axiosApi } from "@/api/axiosConfig.js";
+import { LOGIN } from "@/api/apiEndpoints.js";
+import { REGISTER } from "@/api/apiEndpoints.js";
+import { USER } from "@/api/apiEndpoints.js";
+
+export const login = async (credentials) => {
+  return axiosApi.post(`${LOGIN}`, {
+    user: {
+      email: credentials.email,
+      password: credentials.password,
+    },
+  });
+};
+
+export const register = async (credentials) => {
+  return axiosApi.post(`${REGISTER}`, {
```

Comment:
What I understand since the function register takes credentials argument the params object would have to be created inside of this function (not even sure if it would work), then I would have to create the same object, just not in the request, but still in the function to have access to the argument. Maybe you would like to provide me some example from a real project?